src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -718,37 +716,3 @@ export function fetchFacilitySettings() {
     }
   };
 }
-
-/**
- * Function to retrieve provider information from the provider
- * endpoint when using the v2 api.
- *
- * @param {*} appointment
- */
-export function getProviderInfoV2(appointment) {
-  // Provider information included with v2 provider api call.
-  return async (dispatch, getState) => {
-    const featureVAOSServiceCCAppointments = selectFeatureVAOSServiceCCAppointments(
-      getState(),
-    );
-    if (featureVAOSServiceCCAppointments && appointment.practitioners?.length) {
-      const ProviderNpi = getPreferredCCProviderNPI(appointment);
```

Comment:
Since `getProviderInfoV2` is eliminated, we no longer need `getPreferredCCProviderNPI` found in https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/vaos/services/appointment/index.js#L389